params/state_upgrade_test.go

```diff
@@ -185,3 +185,42 @@ func TestUnmarshalStateUpgradeJSON(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, upgradeConfig, unmarshaledConfig)
 }
+
+func TestSerialize(t *testing.T) {
+	var t0 uint64 = 2
+	var t1 uint64 = 1001
+	config := UpgradeConfig{
+		StateUpgrades: []StateUpgrade{
+			{
+				BlockTimestamp: &t0,
+				StateUpgradeAccounts: map[common.Address]StateUpgradeAccount{
+					common.BytesToAddress(common.Hex2Bytes("0000000000000000000000000000000000000000000000000000000000000050")): StateUpgradeAccount{
```

Comment:
nit: can we create a variable for these addresses instead of hardcoding them into tests?